File: src/main/java/io/lcydev/spawnercleaner/listeners/ChunkLoadListener.java
package io.lcydev.spawnercleaner.listeners;

import io.lcydev.spawnercleaner.SpawnerCleaner;
import io.lcydev.spawnercleaner.handlers.ChunkManager;
import io.lcydev.spawnercleaner.utils.Config;
import org.bukkit.event.Listener;
import org.bukkit.event.world.ChunkLoadEvent;

public class ChunkLoadListener implements Listener {

    public ChunkLoadListener() {
        instance = SpawnerCleaner.inst();
        config = Config.get();
    }

    public void onChunkLoad(ChunkLoadEvent e) {
        ChunkManager.initChunk(e.getChunk());
    }

}

File: src/main/kotlin/io/lcydev/spawnercleaner/listeners/ChunkLoadListener.kt
package io.lcydev.template.listeners

import io.lcydev.template.SpawnerCleaner
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.world.ChunkLoadEvent

class ChunkLoadListener(private val plugin: SpawnerCleaner) : Listener {

    @EventHandler
    fun onChunkLoad(e: ChunkLoadEvent) {
        plugin.chunkManager.initChunk()
    }
}